levelorm/fields.py

```diff
@@ -101,6 +101,20 @@ def serialize(self, buf, value: int):
 	def deserialize(self, buf) -> int:
 		return self.struct.unpack(buf.read(self.struct.size))[0]
 
+class Float(BaseField):
+	'''
+	represents an :class:`float`.
+	stored as a 4-byte float
+	'''
+
+	struct = struct.Struct('f')
```

Comment:
this should probably be `'d'`

```python
float_field = fields.Float()
print('putting', 2**128+1.0)
float_field.serialize(buf, 2**128+1.0)
print(buf.getvalue())
buf.seek(0)
print('got', float_field.deserialize(buf))
```
```
putting 3.402823669209385e+38
b'\x00\x00\x80\x7f'
got inf
```

Reply:
python stores things internally as a "double" (binary64). we should probably add a test that this case round-trips correctly. the binary64 limit is `2**1023`, but python doesn't allow you to even represent that so you get an `OverflowError` well before attempting to hand it to levelorm